sktime/transformers/series_as_features/truncation.py

```diff
@@ -18,7 +16,7 @@ class TruncationTransformer(BaseSeriesAsFeaturesTransformer):
     """MyTransformer docstring
     """
 
-    def __init__(self, lower, upper, dim_to_use=0):
+    def __init__(self, lower=None, upper=None, dim_to_use=0):
```

Comment:
I'm not a big fan of `dim_to_use` as we have the `ColumnTransformer` for that

Reply:
What happens when lower and upper = None? No truncation?

Reply:
when they're none, it picks the shortest length to truncate too.